### ISSUE TITLE
warnings in Django 1.8

### ISSUE THREAD

**Issue by sarfrajimpinge** [omab/python-social-auth#584]
I have upgraded my Django from 1.7 to 1.8 recently and I am getting some warnings regarding social auth.  I have gone through you issue list and found that you are saying that it is solved in #551 .So I have upgraded my Python social auth to latest one but warnings still remain as it is .

```
/home/path_to_project/lib/python3.4/site-packages/social/apps/django_app/default/models.py:29: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.UserSocialAuth doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class UserSocialAuth(models.Model, DjangoUserMixin):

/home/path_to_project/lib/python3.4/site-packages/social/apps/django_app/default/models.py:67: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Nonce doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Nonce(models.Model, DjangoNonceMixin):

/home/path_to_project/lib/python3.4/site-packages/social/apps/django_app/default/models.py:78: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Association doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Association(models.Model, DjangoAssociationMixin):

/home/path_to_project/python3.4/site-packages/social/apps/django_app/default/models.py:91: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Code doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Code(models.Model, DjangoCodeMixin):

```

**Python social auth version**  -> python-social-auth==0.2.3

**Comment by dhilipsiva** [comment]
:+1: For me too.

**Comment by omab** [comment]
Fixed by #550

**Comment by sbhuvanas** [comment]
hi . I am a newbie for Django. I too get the same errors. I am totally confused. can you explain it clearly to solve this issue.
